src/braket/circuits/gate.py

```diff
@@ -49,18 +73,35 @@ def to_ir(self, target: QubitSet) -> Any:
         Returns:
             IR object of the quantum operator and target
         """
+        if self._adjoint:
+            # Use adjoint list of original gate
+            return [elem.to_ir(target) for elem in self.adjoint().adjoint_list()]
+        return self._to_ir(target)
+
+    def _to_ir(self, target: QubitSet) -> Any:
         raise NotImplementedError("to_ir has not been implemented yet.")
 
+    @property
+    def ascii_symbols(self) -> Tuple[str, ...]:
+        """Tuple[str, ...]: Returns the ascii symbols for the quantum operator."""
+        if self._adjoint:
+            return tuple(
+                f"({symbol})^†" if symbol != "C" else symbol for symbol in self._ascii_symbols
```

Comment:
What's the special case here with "C", and why do we think we won't have more?

Reply:
C stands for control (I would've preferred •, but it's a bit late for that). There may be more special characters down the line, but we can deal with that when it happens. I'll add a comment.